src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -59,18 +59,20 @@ public void doIndex(StaplerRequest req, StaplerResponse resp) {
         String payload = null;
         String body = null;
 
-        if ("application/json".equals(type)) {
+        if (type.startsWith("application/json")) {
```

Comment:
According to http://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html, I think it should start with the media type. And as it mentions, it is case- insensitive, I fix it.